src/Microsoft.Azure.WebJobs.Host.Storage/StorageServiceCollectionExtensions.cs

```diff
@@ -36,6 +36,13 @@ public static void AddAzureStorageCoreServices(this IServiceCollection services)
             services.AddSingleton<IConcurrencyStatusRepository, BlobStorageConcurrencyStatusRepository>();
         }
 
+        public static void AddAzureStorageScaleServices(this IServiceCollection services)
```

Comment:
Is this meant to ever be called by a customer?

Reply:
Yes, this will be called from ScaleController, see example here:
https://github.com/Azure/azure-webjobs-sdk/blob/scaler-apis/test/Microsoft.Azure.WebJobs.Host.EndToEndTests/Scale/ScaleHostEndToEndTests.cs#L84

Function host is not supposed to call this.